cmd/kube-scheduler/app/server.go

```diff
@@ -220,7 +220,7 @@ func buildHandlerChain(handler http.Handler, authn authenticator.Request, authz
 	failedHandler := genericapifilters.Unauthorized(scheme.Codecs)
 
 	handler = genericapifilters.WithAuthorization(handler, authz, scheme.Codecs)
-	handler = genericapifilters.WithAuthentication(handler, authn, failedHandler, nil)
+	handler = genericapifilters.WithAuthentication(handler, authn, failedHandler, nil, genericapifilters.NoopCheck)
```

Comment:
this is inconsistent with `cmd/controller-manager/app/serve.go` which passes nil. 

Since you're handling the NoOp thing in `WithAuthentication`, it probably makes more sense to pass nil here.